clang-tools-extra/clang-doc/HTMLGenerator.cpp

```diff
@@ -632,6 +633,19 @@ static std::unique_ptr<HTMLNode> genHTML(const CommentInfo &I) {
     return std::move(ParagraphComment);
   }
 
+  if (I.Kind == "BlockCommandComment") {
+    auto BlockComment = std::make_unique<TagNode>(HTMLTag::TAG_DIV);
+    auto Command = std::make_unique<TagNode>(HTMLTag::TAG_DIV, I.Name);
+    BlockComment->Children.emplace_back(std::move(Command));
+    for (const auto &Child : I.Children) {
+      std::unique_ptr<HTMLNode> Node = genHTML(*Child);
+      if (Node)
+        BlockComment->Children.emplace_back(std::move(Node));
+    }
+    if (BlockComment->Children.empty())
+      return nullptr;
+    return std::move(BlockComment);
```

Comment:
Do these need to be returned via std::move? Iirc the compiler should do the right thing with return values…

The pattern exists elsewhere in this file, it’s not required to change now, but we may want to consider up’s this code to use a more idiomatic style.

Reply:
I could add another pr once this is merged to refactor some of this

Reply:
yeah, lets follow this up in another patch.